docs/whats-new.md

```diff
@@ -4,7 +4,67 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
 
 ## deck.gl v8.0
 
-- `ArcLayer` supports drawing arcs between two 3D positions.
+### Performance
+
+Performance is one of the biggest focus of this update. Layer updates (data change) is 1.5x the speed of the last release, and redraw (viewport change) is 2.5x.
+
+*Benchmark of using 1000 ScatterplotLayers on 2016 Macbook Pro, 2.8 GHz Intel Core i7, 16 GB memory, AMD Radeon R9 M370X 2 GB*
+
+|                   | v7.3  | v8.0  | Change |
+| ----------------- | ----- | ----- | ------ |
+| Initialize        | 298ms | 235ms | -21%   |
+| Update            | 112ms | 72ms  | -36%   |
+| Redraw (CPU Time) | 76ms  | 26ms  | -66%   |
+| Redraw (GPU Time) | 17ms  | 10ms  | -41%   |
+
+### Better Binary Data Support
+
+It is now possible to replace a layer's accessors with binary data attributes. This technique offers the maximum performance in terms of data throughput in applications where a lot of data is loaded and/or frequently updated:
+
+```js
+const data = new Float32Array([
+  0.7, 0.2, 0, 0, 0, 0,
+  0.8, 0.6, 0, 0, 5, 0,
+  0.3, 0.5, 0, 5, 5, 0,
+  0, 0.8, 0.6, 5, 10, 0,
+  0, 0.5, 0.7, 10, 10, 0
+]);
+
+new deck.ScatterplotLayer({
+  id: 'points',
+  data: {
+    length: 5,
+    attributes: {
+      getPosition: {value: data, size: 3, offset: 12, stride: 24},
+      getFillColor: {value: data, size: 3, offset: 0, stride: 24}
+    }
+  }
+  getRadius: 1
+})
+```
+
+This use case is discussed in detail in the [performance developer guide](/docs/developer-guide/performance.md#supply-attributes-directly).
+
+### GPU Data Filter in Aggregation Layers
+
+[DataFilterExtension](/docs/api-reference/extensions/data-filter-extension.md) now supports the following layers from `@deck.gl/aggregation-layers`:
+
+- `HeatMapLayer`
+- `GPUGridLayer`
+- `ScreenGridlayer` (GPU aggregation only)
+- `ContourLayer` (GPU aggregation only)
+- `GridLayer` (GPU aggregation only)
+
+### Other New Features and Improvements
+
+- Render to a frame buffer by specifying the `_framebuffer` prop of [Deck](/docs/api-reference/deck.md).
+- Pick a 3d surface point in the scene by passing `unproject3D: true` to `deck.pickObject` or `deck.pickMultipleObjects`.
+- [ArcLayer](/docs/layers/arc-layer.md) supports drawing arcs between two 3D positions.
+- [TextLayer](/docs/layers/text-layer.md) adds a new prop `backgroundColor`. Picking works when the cursor is over an empty pixel in the text.
```

Comment:
Has this been merged in?  I see this open PR - https://github.com/uber/deck.gl/pull/3903 @xintongxia